.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -29,3 +29,13 @@ body:
       description: Mention the environment details (OS, Database, Userstore, etc..) that the product is running on.
     validations:
       required: false
+  - type: checkboxes
+    id: developer-checklist
+    attributes:
+      label: Developer Checklist
+      description: Please confirm the following when closing the issue.
```

Comment:
shouldn't we check this in the PR level instead? by the time prs are merged and issues are closed, the changes have been already merged. instead it would be better to track this **before** the changes are merged, so that it would be easier for everyone involved, IMO.

Reply:
@pavinduLakshan We have decided to add this checklist to product-is issues since it will be easy to track behavioral changes from issue level rather than tracking them from the PR level. However we are planning to add a checkbox in the PR level to remind the developer to complete this checklist in the relevant product-is issue as well.